<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UtilVectorDialogWOWH</class>
 <widget class="QDialog" name="UtilVectorDialogWOWH">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>174</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vector Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="tLabel">
     <property name="text">
      <string>Title</string>
     </property>
     <property name="buddy">
      <cstring>tEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="tEdit">
     <property name="maxLength">
      <number>255</number>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="dLabel">
     <property name="text">
      <string>Description</string>
     </property>
     <property name="buddy">
      <cstring>dEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="dEdit">
     <property name="maxLength">
      <number>255</number>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QRadioButton" name="saveEntireImage">
     <property name="text">
      <string>Save Entire Image</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QRadioButton" name="saveCurrentView">
     <property name="text">
      <string>Save Current View</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UtilVectorDialogWOWH</receiver>
   <slot>accept()</slot>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UtilVectorDialogWOWH</receiver>
   <slot>reject()</slot>
  </connection>
 </connections>
</ui>
